ScoutSuite/frontend/src/components/Tabs/TabPane/index.js

```diff
@@ -4,48 +4,20 @@ import cx from 'classnames';
 
 import './style.scss';
 
-
 const propTypes = {
-  title: PropTypes.node.isRequired,
-  className: PropTypes.string,
```

Comment:
i use the props you removed in my partials, you'll need to add them back

Reply:
Yes, sorry, that's a merge that went wrong 😢